environments/stg/stg.tfvars

```diff
@@ -634,9 +634,9 @@ frontends = [
   },
   {
     name             = "hmi-apim"
-    custom_domain    = "hmi-apim.aat.platform.hmcts.net"
+    custom_domain    = "hmi-apim.staging.platform.hmcts.net"
```

Comment:
I can't find a CNAME entry for this. Please ensure that the CNAME entries are created before merging this to master.

Reply:
Added, thanks!